airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/InlineEnrollmentCallout.tsx

```diff
@@ -17,7 +18,12 @@ export const EnrollLink: React.FC<PropsWithChildren<unknown>> = ({ children }) =
   );
 };
 export const InlineEnrollmentCallout: React.FC = () => {
-  return (
+  const { userDidEnroll } = useFreeConnectorProgram();
+
+  return userDidEnroll ? (
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    <></>
+  ) : (
```

Comment:
```suggestion
  return userDidEnroll ? null :
```

Reply:
huh, somehow assumed that would change the type; I've run into type issues trying to pass `null` to code that expects jsx, and inappropriately generalized the learning.